packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -38,9 +37,9 @@ module.exports = {
       }
     });
   },
-  runInDevelopEnvironment: function (commands = [], config) {
-    const cmdLine = `${this.getExecString()} develop`;
-    const readyPrompt = "truffle(develop)>";
+  runInREPL: function ({ commands = [], config, replType, displayHost } = {}) {
+    const cmdLine = `${this.getExecString()} ${replType}`;
```

Comment:
This usage suggests `replType` is more of a `command-with-arguments` kind of thing. Maybe `replType` should be `replCommand`, and we can validate it is `develop` or `console`. Of course, you'd have to add a way to send an argument list; maybe `replArgs`? What do you sink?